tests/e2e/analyze_graph/common.py

```diff
@@ -13,6 +13,7 @@
 
 import mgclient
 import pytest
+from gqlalchemy import Memgraph
```

Comment:
Why adding gqla dependency, mgclient is used through all e2e tests?

Reply:
yes, but I wanted to introduce gqlalchemy as an option since it's more flexible for setup and cleanups (best thing would be to totally rewrite everything to gqlalchemy but that's a lot of work)